feat: add document validity check component (#1448)
projectcaluma/ember-caluma: tests/dummy/app/templates/docs/validation.md

# Validation

Built in components to validate Caluma data.

## Usage

### DocumentValidity

> By passing a document to this component checks the validity of all its questions.

Reviewer: It's pretty relevant that it's only validated if this component enters the viewport. Add this and then the docs are alright

Author: Added this line

# Validation

Built in components to validate Caluma data.

## Usage

### DocumentValidity

> By passing a document to this component checks the validity of all its questions.
> The document will only be validated after the component enters the viewport.

Reviewer: ```suggestion
This component validates the passed document for its validity as soon as it enters the viewport and will yield the validity status of the document.
```
